src/lib/components/PostCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import Typography from '@material-ui/core/Typography';
+import CardMedia from '@material-ui/core/CardMedia';
+import { Box, Chip } from '@material-ui/core';
+import { MOCK_DATA, MOCK_DATA_USER } from '../constants/postCard-config';
+import { useStyles } from '../styles/PostCard.styles';
+
+export interface IPostCardProps {
+  mockData: typeof MOCK_DATA;
```

Comment:
```
{
   post: IPost, // imported from src/lib/types.ts
}
```
add missing fileds to the interface if necessary

post should have `author?: IExpert ` as optional field

Reply:
fixed